hooks/post-command

```diff
@@ -25,9 +57,61 @@ plugin_read_list_into_result() {
   [[ ${#result[@]} -gt 0 ]] || return 1
 }
 
+get_codecov_uploader() {
+  case "${OS}" in
+    alpine|linux|macos)
+      # We support these
+      ;;
+    *)
+      error "Your platform (${OS}) is not supported."
+      ;;
+  esac
+
+  if [[ "${OS}" = "alpine" ]]; then
+    apk add gnupg
+  fi
+
+  # One-time step
+  curl \
+      -fSs \
+      --retry 5 \
+      https://keybase.io/codecovsecurity/pgp_keys.asc \
+      | gpg \
+          --no-default-keyring \
+          --keyring trustedkeys.gpg \
+          --import
+  local file
+  for file in codecov codecov.SHA256SUM codecov.SHA256SUM.sig
+  do
+    local local_path="${TMP_DIR}/${file}"
+    if [[ ! -e "${local_path}" ]]; then
+      >&2 echo "Local path will be: ${local_path}"
```

Comment:
```suggestion
      debug "Local path will be: ${local_path}"
```

Reply:
Fixed in https://github.com/joscha/codecov-buildkite-plugin/pull/19/commits/827e857d4ed7856630cfafd8340cdbf66f0298a3.